argo-workflows/nautobot-interface-sync/sensors/nautobot-interface-update-sensor-int-sync.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Sensor
+metadata:
+  finalizers:
+  - sensor-controller
+  labels:
+    argocd.argoproj.io/instance: argo-events
+  name: nautobot-interface-update-int-sync
+  namespace: argo-events
+spec:
+  dependencies:
+  - eventName: nautobot
+    eventSourceName: nautobot-webhook
+    name: nautobot-dep
+    filters:
+      dataLogicalOperator: "and"
+      data:
+        - path: "body.event"
+          type: "string"
+          value:
+            - "updated"
+        - path: "body.model"
+          type: "string"
+          value:
+            - "interface"
```

Comment:
I am not sure about this one. If I understand correctly, this filter listens for all `update` events where the `interface` attribute has changed. Now, when this Sensor is activated, it triggers a bunch of workflows that will presumably discover more interfaces for a device and as a result, trigger more webhooks, that would unnecessarily trigger Sensor and more workflow instances again. Is there any way we could filter this down to only trigger once per device?

Reply:
as discussed, we will only react on changes to OBM interfaces, focus is primarily on getting interfaces in when device is created in Nautobot